src/main/webapp/app/overview/course-overview.service.ts

```diff
@@ -75,6 +76,7 @@ export class CourseOverviewService {
 
     faBullhorn = faBullhorn;
     faHashtag = faHashtag;
+    faLock = faLock;
```

Comment:
We could improve the data encapsulation here. At least making it read only would already be a first step.